model/src/main/scala/explore/model/ProposalDetails.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats._
+import explore.model.enum._
+import lucuma.core.model.StandardUser
+import monocle.macros.Lenses
+
+@Lenses
+final case class ProposalDetails(
+  title:         String,
+  pi:            StandardUser,
+  proposalClass: ProposalClass,
+  category:      TacCategory,
+  toOActivation: ToOActivation,
+  keywords:      List[Keyword],
```

Comment:
I guess this could be a `Set`?

Reply:
Good point. I think I need to make some changes to `EnumViewMultipleSelect` to handle that. It would make sense that any time we use a multiple select that the result is a `Set`.